app/mailers/notify_mailer.rb

```diff
@@ -0,0 +1,33 @@
+class NotifyMailer < GovukNotifyRails::Mailer
+  BLANK_TEMPLATE_ID="36555d23-b0e0-4c10-9b85-9c79c98eb1fe".freeze
+  CONFIRMATION_TEMPLATE_ID="a44bc231-d779-41d8-a5e0-180497dfa711".freeze
+
+  include Devise::Controllers::UrlHelpers
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.notify_mailer.test_email.subject
+  #
+  def test_email
+    set_template(BLANK_TEMPLATE_ID)
+    @greeting = "Hi"
+
+    set_personalisation(
+      subject: 'This is just a test',
+      name: "Brett McHargue",
+      body: "this is the body?"
+    )
+    mail to: "brett.mchargue@education.gov.uk"
```

Comment:
As this is a test email account, should we use a test email. Something like `someone@example.com`. It shouldn't hit a mailbox, and only be caught in the Notify test system, so it doesn't need to have a real mailbox at the end. And using `example.com` in testing is good practice I think because it makes it a little harder to accidentally spam an actual email address in testing.

Reply:
I used it initially to see if Notify would actually send an email and it needs to be one of the team members or on a whitelist.  I think it would be good to keep around - once we are using a live API key we will want to test that it works with a real email address.